src/api/spec/README.md

```diff
@@ -187,14 +185,11 @@ You should only use shared examples where you have the exact same functionality
 Otherwise these tests get fast hard to refactor and review.
 In our experience, shared examples are used mainly for controllers. Since models are pretty different from each other, they (usually) do not share much logic.
 
-### Travis
-We use [travis-ci](https://travis-ci.org/) for continues integration.
+### Continues Integration
```

Comment:
~~Continues~~ -> Continuous

Reply:
There's the same typo one line below too

Reply:
Argh right